highlighter/tox.ini

```diff
@@ -3,6 +3,8 @@ envlist = py27, py3, flake8
 skipsdist = true
```

Comment:
A curiosity, what are you using the `tox.ini` file for? Do we need this, it appears that the only way to run your code is via the browser console at the moment.

Reply:
I now remember where this tox.ini file came from! You added lines 6 and 7 so that I would be able to run this on my virtual box/linux os. Still not sure how it went in though since I added, committed orphanage files only. Still waiting for checks to be completed so I can delete this file and add refactor issue in tracker.

Reply:
Ah yes, that's right. Let's leave this in place.

Reply:
Good memory on your part!